micrometer-spring-legacy/src/samples/java/io/micrometer/spring/samples/PrometheusSample.java

```diff
@@ -25,4 +28,27 @@
     public static void main(String[] args) {
         new SpringApplicationBuilder(PrometheusSample.class).profiles("prometheus").run(args);
     }
+
+    @Bean
+    public Tracer tracer() {
+        Tracer tracer = new com.uber.jaeger.Configuration(
```

Comment:
why do you need to use uber's tracer.. I thought opentracing has a test variant?

Reply:
This is in the samples and won't be checked in long term. 

It is essentially a spring boot app that I run to play around and explore. Thanks for the point about using the test variant in the unit tests.

I left it in the PR to demonstrate if anyone wanted to check it out and try running it.